main.go

```diff
@@ -20,9 +22,21 @@ import (
 )
 
 var (
+	// Platforms maps all possible media-types/platforms to it's IDs used in a request
+	Platforms = map[int]string{
```

Comment:
The original reasoning, why this map was in the collector was a separation of concerns, as only the instance that actually works with those definitions should be concerned with them. But I think that the added functionality outweighs this reasoning.. even though this validation and map-creation might as well be moved to the collector to be done upon creation, right?
Anyways that doesn't stop me approving this PR for the added functionality :)

Reply:
I think wiring up the platforms in `main` would even help with separation of concerns here: the collector would simply deal with collecting metrics for the platforms passed to it and the main would do the handling of the environment variables and the creation of the platforms map.
This way, things would be easier for testing as the map of platforms to crawl could be created in a unit test and not hidden inside the implementation. 😉

Reply:
Fair enough :+1: Merging :)